lib/api/apiUtils/object/checkReadLocation.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * checkReadLocation - verify that a bucket's default read location exists
+ * a specified read data locator
+ * @param {Config} config - Config object
+ * @param {string} locationName - location constraint
+ * @param {string} objectKey - object key
+ * @param {string} bucketName - bucket name
+ * @return {Object | null} return object if location exists; otherwise, null
```

Comment:
minor: I suggest a more descriptive comment about the returned value, e.g. returns the data storage location?